src/worker/core/season/awards.ts

```diff
@@ -303,15 +304,20 @@ const saveAwardsByPlayer = async (
 			conditions,
 		);
 	}
-
 	const pids = Array.from(new Set(awardsByPlayer.map(award => award.pid)));
 	for (const pid of pids) {
-		const p = await idb.cache.players.get(pid);
+		let p = await idb.cache.players.get(pid);
+		if (!p) {
+			const p2 = await idb.getCopy.players({
+				pid: pid,
+			});
+			p = p2 as PlayerFiltered;
```

Comment:
`PlayerFiltered` is `any`, so might as well just call this `any` if that's what needs to be done. I feel like it should work to just make line 311.

this should work:

```
			p = await idb.getCopy.players({
				pid: pid,
			}); 
```

...except my return value from `getCopy.players` is wrong, should be undefined rather than void. If changing that causes other problems, don't worry about it, I can deal with this :)